app/cdap/components/GridTable/index.tsx

```diff
@@ -80,9 +90,9 @@ export default function GridTable() {
           const visualization = objectQuery(res, 'insights', 'visualization') || {};
 
```

Comment:
add expected type for visualization

Reply:
Did not change as res is coming from API